Source/Blazorise.AntDesign/Select.razor.cs

```diff
@@ -178,14 +215,68 @@ static IEnumerable<TValue> ParseMultipleValues( string csv )
 
         protected string InputElementId { get; set; } = IDGenerator.Instance.Generate;
 
+        /// <summary>
+        /// Gets component items.
+        /// </summary>
+        public Dictionary<TValue, RenderFragment> Items => items;
+
+        /// <summary>
+        /// Gets the selected items.
+        /// </summary>
+        protected IEnumerable<RenderFragment> SelectedItems
+        {
+            get
+            {
+                var list = new List<RenderFragment>();
+
+                foreach ( var selectedValue in SelectedValues )
+                {
+                    list.Add( items[selectedValue] );
+                }
+
+                return list;
+            }
+        }
+
+        protected RenderFragment SelectedItem
+        {
+            get
+            {
+                return
+                SelectedValue != null
+                ? items.Count > 0
+                ? items[SelectedValue]
+                : null
+                : null;
+            }
+        }
+        /// <summary>
+        /// Gets or sets values comparer.
+        /// </summary>
+        [Parameter]
+        public IEqualityComparer<TValue> EqualityComparer
```

Comment:
Can you please tell me what is the reason for EqualityComparer as a parameter in AntDesign since it doesn't exist in base Select component? AntDesign Select is just a custom implementation of the base component and it shoud not be used directly.

Reply:
Idea was to use not primitive for Value values but it is not supported by Blazor. So default comparer is enough. I'll remove  this parameter.